server/app/services/cloud/aws/SimpleEmail.java

```diff
@@ -30,6 +32,26 @@ public final class SimpleEmail {
   public static final String AWS_SES_SENDER_CONF_PATH = "aws.ses.sender";
   private static final Logger logger = LoggerFactory.getLogger(SimpleEmail.class);
 
+  private static final Histogram emailExecutionTime =
```

Comment:
**nitpick:** use SCREAMING_SNAKE_CASE for constants

Reply:
Updated, thanks!